tests/helpers/AbstractTestClient.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver;
+
+use Symfony\Component\BrowserKit\Response;
+use Symfony\Component\HttpKernel\Kernel;
+
+if (Kernel::VERSION_ID >= 40300) {
+    require_once __DIR__ . '/AbstractClientForAbstractBrowser.php';
+} else {
+    require_once __DIR__ . '/AbstractClientForClient.php';
+}
+
+abstract class AbstractTestClient extends AbstractClient
+{
+    protected $nextResponse = null;
```

Comment:
Since we're using PHP 5.4 now we can move all these methods a trait and include it in above defined `AbstractClient` classes.

Reply:
we are not. I reverted that change, until drivers themselves stop using the testsuite on PHP 5.3